x.py

```diff
@@ -38,6 +38,16 @@
 
 default_alpine_version = "3.12"
 
+windows_versions = [
+    ("1709", "16299"),
+    ("1803", "17134"),
+    ("1809", "17763"),
+    ("1903", "18362"),
+    ("1909", "18362"),
+    ("2004", "19041"),
+    ("20H2", "19041"),
```

Comment:
Do other official images cover this many Windows revisions?

Reply:
This list includes all of the non-duplicated Windows revisions that are in the full tag listing of the [Windows Server Core Docker Hub page](https://hub.docker.com/_/microsoft-windows-servercore).

Reply:
It looks like golang's currently just provide 1809 and ltsc2016: https://hub.docker.com/_/golang

Reply:
Limiting to just 1809 is easy enough, but I don't think we can support 1607/ltsc2016 as the Visual Studio installer doesn't have a matching/compatible Windows 10 SDK version available. We can only support up to 1809 using Github Actions or Travis because that is the latest version of Windows Server they support. I have not looked into Azure's support.

Reply:
Okay, let's just go with 1809 for now.